translations/messages+intl-icu.en.yaml

```diff
@@ -72,3 +74,69 @@ blog.title: Tech, Teams and Tales
 blog.subtitle: echoes from baksla.sh
 blog.no_article_yet: There is no article yet, come back later!
 blog.article.more_from_the_blog: More from <0>The Blog</0>
+
+team.seo.title: Meet the Team
+team.seo.description: Discover the baksla.sh's crew at work
+team.title: Shhhht! The crew is at work 🤫
+team.subtitle: Discover the baksla.sh team members while they're working
+team.click_on_member: 💡 Click on a member to discover its full profile.
+team.modal.nice: Nice!
+team.reviewers: Reviewers
+team.review.twig_alias_1: Could we alias this import to <0>Twig</0> to make it a little more understandable?
+team.review.twig_alias_2: Agreed! "Environment" doesn't mean a lot to me as well.
+team.review.twig_alias_3: Another idea could be to extend the <0>AbstractController</0>, WDYT?
+team.review.infra_1: Always use interfaces instead of concrete implementations.<0/>It takes advantage of <1>polymorphism</1> and reduce coupling between classes.
+team.review.infra_2: 👍 Plus, the <0>MemberRepository</0> belongs the "Domain" layer.<1/>We must rely as much as possible on code from that layer.
+team.review.infra_3: If you want to have a specific implementation, you can use the <0>"Autowire" attribute</0>.
+team.review.get_method_1: "While it works, it's not the best way to achieve that. You must use instead leverage <0>#[Route(methods: ['GET'])]</0>."
+team.review.get_method_2: He's right, while it make it declarative, it allows us to see that only <0>GET</0> is allowed when using <0>debug:router</0>.
+team.review.get_method_3: Same with <0>router:match</0> actually!
+team.review.cache_1: Shouldn't we cache the result somehow?
+team.review.cache_2: 💯
+
+team.bio.arnaud-de-abreu: |-
```

Comment:
We still miss:
- [ ] `team.bio.enzo-santamaria`
- [ ] `team.bio.jules-pietri`
- [ ] `team.bio.yazid-hassani`

Reply:
Yazid's bio will be handled in a later PR.